### PR TITLE
Swope gh changes

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: global_hub/global_hub_components.adoc [hunk 1]
```diff
@@ -1,7 +1,8 @@
 [#global-hub-integrating-existing-components]
 = Integrating existing components
 
-{global-hub-cap} requires the middleware components Kafka and PostgreSQL, along with Grafana as the observability platform to provide the policy compliance view. {global-hub-cap} provides versions of Kafka, PostgreSQL, and Grafana. You can also integrate your own existing Kafka, PostgreSQL, and Grafana.
+The {global-hub} requires middleware components, Kafka and PostgreSQL, along with Grafana as the Observability platform to provide the policy compliance view. The {global-hub} provides versions of Kafka, PostgreSQL, and Grafana. You can also integrate your own existing Kafka, PostgreSQL, and Grafana.
+//revisit "components"
```
Comment (swopebe) [inline global_hub/global_hub_components.adoc]: discuss this.

File: global_hub/global_hub_components.adoc [hunk 2]
```diff
@@ -41,6 +42,7 @@ oc create secret generic multicluster-global-hub-transport -n multicluster-globa
 . If automatic topic creation is configured on your Kafka instance, then skip this step. If it is not configured, create the `spec`, `status`, and `event` topics manually. 
 
 . Ensure that the Kafka user has the permission to read data from the topics and write data to the topics.
+//we switch to the Kafka user here...is that because it wasn't changed or there is another user?
```
Comment (swopebe) [inline global_hub/global_hub_components.adoc:44]: discuss this

File: global_hub/global_hub_grafana_alerts.adoc [hunk 3]
```diff
@@ -1,27 +1,29 @@
...
+* Cron Job failed: This alert watches the cron jobs that are described in xref:../global_hub/global_hub_config_cronjobs.adoc#global-hub-configuring-cronjobs[Configuring the cron jobs] for failed events. There are two rules in this alert:
 +
-- Local compliance job failed. If this alert rule creates notifications, it means the local compliance status synchronization job failed. It might cause the data in the `history.local_compliance` table to be lost. Run the job manually, if necessary.
-- Data Retention Job Failed. If this alert rule starts creating notifications, it means the data retention job failed. You can run it manually.
+- Local compliance job failed: If this alert rule creates notifications, it means the local compliance status synchronization job failed. It might cause the data in the `history.local_compliance` table to be lost. Run the job manually, if necessary.
+- Data retention job failed: If this alert rule starts creating notifications, it means the data retention job failed. You can run it manually.
```
Comment (swopebe) [inline global_hub/global_hub_grafana_alerts.adoc:21]: discuss

File: global_hub/global_hub_requirements.adoc [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+|Platform | Supported for global hub cluster | Supported for managed hub cluster
+
+|{product-title-short} on Arm | No | Yes |
+{product-title-short} on IBM Z | No | Yes
+|{product-title-short} on IBM Power Systems | No | Yes
+
+** Kafka 3.3 and later 3.3.x releases. 
+//We have this line here-it was already here, and we have a whole components topic...I don't think we need both and I think we need to choose where and how we present "components" for SEO, we have it here with "main components," something different than we have in the components file (middleware), which is confusing.
+** PostgreSQL version 13 and later 13.x releases.
+//We have this line here, and we have a whole components topic...I don't think we need both and I think we need to choose where and how we present "components" for SEO, we have it here with something different than we have in the components file, which is confusing.
```
Comment (swopebe) [inline global_hub/global_hub_requirements.adoc]: discuss

File: global_hub/global_hub_architecture.adoc [hunk 5]
```diff
@@ -1,35 +1,41 @@
 [#global-hub-architecture]
-= {global-hub-cap} architecture
+= {global-hub} architecture
```
Comment (cadawson) [inline global_hub/global_hub_architecture.adoc:2]: The {global-hub} architecture? Maybe lowercase is ok? Maybe spell this out for initial cap?

File: global_hub/global_hub_architecture.adoc [hunk 6]
```diff
@@ -1,35 +1,41 @@
...
 
-The {global-hub} consists of several components, which are used to access and manage your hub clusters. Learn more about the architecture in the following sections.
+The {global-hub} consists of components that are used to access and manage your hub clusters.
```
Comment (cadawson) [inline global_hub/global_hub_architecture.adoc]:
```suggestion
The {global-hub} consists of the following components that are used to access and manage your hub clusters:
```

File: global_hub/global_hub_architecture.adoc [hunk 7]
```diff
@@ -1,35 +1,41 @@
...
-* <<global-hub-agent,{global-hub-cap} agent>>
-* <<global-hub-visualizations,{global-hub-cap} visualizations>>
+* <<global-hub-operator,{global-hub} operator>> 
+* <<global-hub-manager,{global-hub} manager>>
+* <<global-hub-agent,{global-hub} agent>>
+* <<global-hub-visualizations,{global-hub} visualizations>>
 
 [#global-hub-operator]
-== {global-hub-cap} operator
+== {global-hub} operator
```
Comment (cadawson) [inline global_hub/global_hub_architecture.adoc]: The {global-hub} operator? Most of our sections are initial cap, but there are a few exceptions. We can probably leave this. I thought about removing {global-hub}, but for search purposes, I think it is helpful. Similar for the rest of these.

File: global_hub/global_hub_architecture.adoc [hunk 8]
```diff
@@ -1,35 +1,41 @@
...
 
 The {global-hub} operator contains the components of {global-hub}. The operator deploys all of the required components for global multicluster management. The components include `multicluster-global-hub-manager`, `multicluster-global-hub-grafana`, and provided versions of `Kafka` and `PostgreSQL` in the {global-hub} cluster and `multicluster-global-hub-agent` in the managed hub clusters.
 
 The operator also leverages the `manifestwork` custom resource to deploy the Red Hat Advanced Cluster Management for Kubernetes operator on the managed cluster. After the Red Hat Advanced Cluster Management operator is deployed on the managed cluster, the managed cluster becomes a standard {product-title-short} cluster. This hub cluster is now a managed hub cluster.
 
 [#global-hub-manager]
-== {global-hub-cap} manager
+== {global-hub} manager
```
Comment (cadawson) [inline global_hub/global_hub_architecture.adoc]: The {global-hub} manager?

File: global_hub/global_hub_architecture.adoc [hunk 9]
```diff
@@ -1,35 +1,41 @@
...
 
 The {global-hub} manager is used to persist the data into the `postgreSQL` database. The data is from Kafka transport. The manager also posts the data to the Kafka transport, so it can be synchronized with the data on the managed hub clusters.
 
 [#global-hub-agent]
-== {global-hub-cap} agent
+== {global-hub} agent
```
Comment (cadawson) [inline global_hub/global_hub_architecture.adoc]: The {global-hub} agent?

File: global_hub/global_hub_architecture.adoc [hunk 10]
```diff
@@ -1,35 +1,41 @@
...
 
 The {global-hub} agent runs on the managed hub clusters. It synchronizes the data between the {global-hub} cluster and the managed hub clusters. For example, the agent synchronizes the information of the managed clusters from the managed hub clusters to the {global-hub} cluster and synchronizes the policy or application from the {global-hub} cluster to the managed hub clusters.
 
 [#global-hub-visualizations]
-== {global-hub-cap} visualizations
+== {global-hub} visualizations
```
Comment (cadawson) [inline global_hub/global_hub_architecture.adoc]: The {global-hub} visualizations?

File: global_hub/global_hub_components.adoc [hunk 11]
```diff
@@ -1,11 +1,11 @@
 [#global-hub-integrating-existing-components]
 = Integrating existing components
 
-{global-hub-cap} requires the middleware components Kafka and PostgreSQL, along with Grafana as the observability platform to provide the policy compliance view. {global-hub-cap} provides versions of Kafka, PostgreSQL, and Grafana. You can also integrate your own existing Kafka, PostgreSQL, and Grafana.
+The {global-hub} requires middleware components, Kafka and PostgreSQL, along with Grafana as the Observability platform to provide the policy compliance view. The {global-hub} provides versions of Kafka, PostgreSQL, and Grafana. You can also integrate your own existing Kafka, PostgreSQL, and Grafana.
 
-* <<integrating-existing-kafka,Integrating an existing verison of Kafka>>
-* <<integrating-existing-postgresql,Integrating an existing verison of PostgreSQL>>
-* <<integrating-existing-grafana,Integrating an existing verison of Grafana>>
+* <<integrating-existing-kafka,Integrating an existing version of Kafka>>
```
Comment (cadawson) [inline global_hub/global_hub_components.adoc:6]: Nice catch here.

File: global_hub/global_hub_components.adoc (hunk 2 above)
Comment (cadawson) [reply global_hub/global_hub_components.adoc:44]: Fair point. I read this as: you are making sure that the global hub user that accesses the existing Kafka data can read data from and write data to on the Kafka topics. That could be better.

**Review by cadawson — APPROVED** [review]
Looks ok to me. A couple of comments that might or might not apply. Thanks!

File: global_hub/global_hub_architecture.adoc (hunk 5 above)
Comment (swopebe) [reply global_hub/global_hub_architecture.adoc:2]: makes sense, good catch

**Pull request merged**